projects/driver_controls_pedal/src/power_state_indicator.c

```diff
@@ -0,0 +1,34 @@
+#include "power_state_indicator.h"
+#include "can.h"
+#include "can_msg_defs.h"
+#include "can_unpack.h"
+#include "exported_enums.h"
+#include "pc_input_event.h"
+
+static StatusCode prv_handle_power_state(const CanMessage *msg, void *context,
+                                         CanAckStatus *ack_reply) {
+  uint8_t state = 0;
+  CAN_UNPACK_POWER_STATE(msg, &state);
+
+  switch (state) {
+    case EE_POWER_STATE_IDLE:
+      event_raise_priority(EVENT_PRIORITY_NORMAL, INPUT_EVENT_PEDAL_POWER_STATE_OFF, 0);
```

Comment:
You can refactor this switch statement to merge the `EE_POWER_STATE_IDLE` and default case by making `EE_POWER_STATE_IDLE` fall through to the default case